server/controllers/recommendation/searchPpudController.test.ts

```diff
@@ -9,11 +15,106 @@ describe('get', () => {
       },
     })
     const next = mockNext()
-    await ppcsConsiderRecallController.get(mockReq(), res, next)
+    await searchPpudController.get(mockReq(), res, next)
 
     expect(res.locals.page).toEqual({ id: 'searchPpud' })
     expect(res.render).toHaveBeenCalledWith('pages/recommendations/searchPpud')
 
     expect(next).toHaveBeenCalled()
   })
 })
+
+describe('post', () => {
+  it('post with some results', async () => {
+    const results = [
+      {
+        id: '4F6666656E64657269643D313731383138G725H664',
+        croNumber: '123456/12A',
+        nomsId: 'JG123POE',
+        firstNames: 'John',
+        familyName: 'Teal',
+        dateOfBirth: '2000-01-01',
+      },
+    ]
+
+    ;(searchPpud as jest.Mock).mockResolvedValue({ results })
+
+    const basePath = `/recommendations/1/`
+    const req = mockReq({
+      params: { recommendationId: '1' },
+      body: {
+        crn: 'X123',
+        croNumber: '678',
+        nomsNumber: '234',
+        surname: 'McBoverin',
+        dateOfBirth: '1980-10-02',
+        fullName: 'Ducko',
```

Comment:
Do we use fullName? It's not required for PPUD Automation.

Reply:
its for how the no results found page.